analyzer/evmtokens/evm_tokens.go

```diff
@@ -147,11 +147,13 @@ func (m Main) processBatch(ctx context.Context) (int, error) {
 	defer cancel()
 	group, groupCtx := errgroup.WithContext(ctxWithTimeout)
 
-	batch := &storage.QueryBatch{}
+	var batches []*storage.QueryBatch
 
 	for _, st := range staleTokens {
 		// Redeclare `st` for unclobbered use within goroutine.
 		staleToken := st
+		batch := &storage.QueryBatch{}
+		batches = append(batches, batch)
```

Comment:
It's particularly nice for debugging, run comparisons, test writing etc, if the sql statements are ordered deterministically.
Maybe you can sort `staleTokens` before this for loop? Or probably even better, make `getStaleTokens()` return ordered results by adding an `ORDER BY token_address` to the underlying [db query](https://github.com/oasisprotocol/oasis-indexer/blob/a05eb962b1588e11ad495d2e907e94bdf3db075c/analyzer/queries/queries.go#L318-L318).

Reply:
hm test determinism. I'm going to open an issue for that.

Reply:
#345